### PR TITLE
Complete terminal user interface

### PR DESCRIPTION
**Pull request by Vgonma**
## Added UI that lets you:
  - List all books.
  - List all people.
  - Create a book.
  - Create a person (teacher or student).
  - Create a rental.
  - List all rentals for a given person's id.

## Additional functionality
  - Asks to fill in the parameters for the required operation if needed.
  - Has a way to quit the app.

### CODE REVIEW

File: main.rb [hunk 1]
```diff
@@ -1,33 +1,65 @@
...
+def main
+  option = -1
+  app = App.new
+  until option.zero?
+    main_menu
+    option = gets.chomp.to_i
+    puts ''
+    run_menu(app, option) unless option.zero?
+  end
+  puts 'Closing...'
```
Comment (mwanawabangona) [inline main.rb]: - [ ] `required`
- [x] Your console app should behave in the same way as it is in the example.
- [x] Please change the exit message to `hank you for using this app` just as the example 🥂

**Review by mwanawabangona — CHANGES_REQUESTED** [review]
Hi @Vgonma ,

Good job so far!

There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
* ✅  No linter errors :heavy_check_mark:
* ✅  Repo looks professional :briefcase:
* ✅  Good quality commits


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: main.rb (hunk 1 above)
Comment (Vgonma) [reply main.rb]: @mwanawabangona The console behaves exactly the same which is not part of the requirements. It does leave a message that tells you that it is closing same as the example.

**Review by rustam-tolipov — APPROVED** [review]
Hi @Vgonma, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Developer-512px-10

## Highlights
1. No Linters errors 💯
2. All requirements are complete 💯
3. A medal for clean code 🥇 

## Optional suggestions
No optional suggestions :)

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**